draftjs_exporter/options.py

```diff
@@ -30,37 +32,38 @@ def __ne__(self, other):
         return not self == other
 
     @staticmethod
-    def for_block(block_map, type_):
-        if type_ not in block_map:
-            raise ConfigException('Block "%s" does not exist in block_map' % type_)
-
-        block = block_map.get(type_)
+    def for_kind(kind_map, type_, fallback_key):
+        """
+        Create an Options object from any mapping.
+        """
+        if type_ not in kind_map:
+            if fallback_key not in kind_map:
+                raise ConfigException('"%s" is not in the config and has no fallback' % type_)
+
+            config = kind_map[fallback_key]
+        else:
+            config = kind_map[type_]
 
-        if isinstance(block, list):
-            raise ConfigException('Block "%s" uses unsupported list-style config' % type_)
-        elif isinstance(block, dict):
-            if 'element' not in block:
-                raise ConfigException('Block "%s" does not define an element' % type_)
+        if isinstance(config, list):
```

Comment:
This `list` check is only meant as a deprecation helper. In the interest of moving fast this week, and because I'm not sure it was even a good idea to start with, I'll get rid of it and move on.

I get the feeling your comment still holds true for the `dict` check, but let's wait for someone to shoot themselves in the foot before we fix that (the documentation is getting better, so I'm willing to think it will be good enough in this respect).

One more thing – at this stage the element can be two types of things:

- Anything that the `DOM.create_element` function accepts (tag name, function, class, class instance)
- A dict wrapping the above, with an `element` key and optionally other keys to define wrapper & props.